lib/Dialect/Openfhe/Transforms/ConfigureCryptoContext.cpp

```diff
@@ -132,16 +132,30 @@ LogicalResult convertFunc(func::FuncOp op, int64_t mulDepth) {
   ImplicitLocOpBuilder builder =
       ImplicitLocOpBuilder::atBlockEnd(module.getLoc(), module.getBody());
 
+  // get mulDepth from function argument ciphertext type
```

Comment:
Can be done as a follow-up issue/PR, but at this point it's not really a "mulDepth" anymore, more of "number of RNS limbs", so it might make sense to rename it (and also in the corresponding OpenFHE dialect op?)

Reply:
OpenFHE API used the name `SetMultiplicativeDepth` so I think the naming here is in accordance with that.